sdk/python/tests/conftest.py

```diff
@@ -173,7 +173,15 @@ def simple_dataset_2() -> pd.DataFrame:
 
 def start_test_local_server(repo_path: str, port: int):
     fs = FeatureStore(repo_path)
-    fs.serve("localhost", port, no_access_log=True)
+    fs.serve(
```

Comment:
Yes, those are not defined as optional for feature_store.serve() method.

Reply:
Other parameters have [default values](https://github.com/feast-dev/feast/blob/master/sdk/python/feast/feature_store.py#L2514) set. You don't need to provide them, only `host` and `port` are mandatory.

Reply:
@tokoko - I see that feature_store is modified [recently](https://github.com/feast-dev/feast/commit/fa744380ad48ee394a05b2f600db5fb0a99c03aa#diff-ff4e2a65c8c29b7971edc77a61a7e970d410d77d9f173f72714d1bb49e2509a2R2451) after i started working on this change. The default values have been added to original API so that I can revert this change.

Reply:
reverted this change, please take a look.